bench/vector_tile.js

```diff
@@ -56,9 +56,9 @@ Tile.Feature.read = function (pbf, end) {
 };
 Tile.Feature._readField = function (tag, obj, pbf) {
     if (tag === 1) obj.id = pbf.readVarint();
-    else if (tag === 2) obj.tags = pbf.readPackedVarint();
+    else if (tag === 2) obj.tags = pbf.readPackedVarint(obj.tags);
```

Comment:
Edge case ofor compiling a proto file with the *new* version of pbf and then running with the old version of pbf (that doesn't manipulate in place).

I can only picture this being a problem if the compiled files are distributed separately from pbf (which seems unlikely).